docs/competition/team-support/helpdesk.md

```diff
@@ -89,13 +87,11 @@ Some teams may wish to work on their robot on Saturday night and will therefore
 
 Some teams have been granted permission to keep the kit for a little while after the competition. To be able to make use of the kit they will require batteries and stuff. They **MUST NOT** be given any of these items from helpdesk. They will receive these items at the 'Kit Return Desk' downstairs when they leave. All teams must go to the Kit Return Desk, regardless of whether they are returning their kit or not as that is where teams keeping kit will receive their batteries and stuff.
 
-### Handling requests for velcro
-
-Teams may request a piece of velcro to use on their robot for the purpose of attaching the robot badges. As we have a limited supply of velcro, please follow the instructions on the velcro measurements/instruction sheet. Make sure that you cross off their TLA once they have a piece of velcro so that they can't keep coming back for more.
+### Handling requests for flag fittings
 
-### Handling requests for sample robot badge
+Teams may request a pipe connector to use on their robot for the purpose of attaching the robot flags. As we have a limited supply, make sure that you only let each team only take one, so that they can't keep coming back for more.
 
-The robots have 'robot badges' on them during competition matches. A team may ask for a sample badge to check for size. Helpdesk has a single sample badge to lend out to teams. Teams **MUST** return this badge to helpdesk ASAP. When handing it out take note of the team's TLA and chase them up if they haven't returned it in 5 mins.
+Also, please show teams how to remove the fittings, as they are very strong and need to be released in the correct way (Press down on the center ring)
 
 ### Frequently Asked Questions
```

Comment:
Maybe drop the point about tablets, or note that we didn't ship them this year?

Reply:
Fixed in ce6f72ed576c348d2e8c6c890159f989cb987b78